_includes/devdoc/dash-core/rpcs/rpcs/listunspent.md

```diff
@@ -133,17 +133,22 @@ the *spendable* field in the results described below.
   p: "Required<br>(exactly 1)"
   d: "The number of PrivateSend rounds"
 
+- n: "→ →<br>`safe`"
+  t: "bool"
+  p: "Required<br>(exactly 1)"
+  d: "*Added in Bitcoin Core 0.15.0*<br><br>Whether this output is considered safe to spend. Unconfirmed transactions from outside keys and unconfirmed replacement transactions are considered unsafe and are not eligible for spending by `fundrawtransaction` and `sendtoaddress`."
```

Comment:
We don't have `unconfirmed replacement transactions`

Reply:
Good point. I just copied this directly from the RPC help output and overlooked that. :wink: 
https://github.com/dashpay/dash/pull/2727/files#diff-df7d84ff2f53fcb2a0dc15a3a51e55ce